docs/rules/test-case-property-ordering.md

```diff
@@ -0,0 +1,56 @@
+# enforce ordering of keys in test cases (test-case-property-ordering)
+
+This rule enforces that the properties of RuleTester test cases are arranged in a consistent order.
+
+## Rule Details
+
+### Options
+
+This rule has an array option:
+
+* `["code", "output", "options", "parserOptions", "errors"]` (default): The properties of a test case should be placed in a consistent order.
+
+Examples of **incorrect** code for this rule:
+
+```js
+/* eslint eslint-plugin/test-case-property-ordering: ["error",
+  ["code", "output", "options", "parserOptions", "errors"]
+] */
+
+// valid;
+{
+  code: "foo",
+  output: "bar",
+  options: ["baz"],
```

Comment:
I think these documentation examples are switched around -- the valid samples are in the block that says "Examples of incorrect code", and the invalid samples are in the block that says "Examples of correct code".